raster_cutter.py

```diff
@@ -430,37 +499,45 @@ def delete_world_file(directory_url):
     if os.path.exists(worldfile_path):
         os.remove(worldfile_path)
     else:
-        raise Exception("The file does not exist")
+        raise Exception('The file does not exist')
 
 
 def get_worldfile_url_from_dir(directory_url):
-    index = directory_url.find(".")
+    index = directory_url.find('.')
     if index != -1:
         worldfile_path = directory_url[:index]
-        worldfile_path += ".wld"
+        worldfile_path += '.wld'
     else:
-        raise Exception("Could not find . in path")
+        raise Exception('Could not find . in path')
```

Comment:
This error is technically correct, but can we be more specific, which the underlying problem is (even from the code I don't know, what the issue is). Does this test for path exists and path isdir?